indico/modules/events/timetable/client/js/Timetable.tsx

```diff
@@ -49,36 +50,30 @@ export default function Timetable() {
 
   const useWeekView = false;
 
-  const minHour = Math.max(
-    0,
-    useWeekView
-      ? Math.min(
-          ...Object.values(entries)
-            .flat()
-            .map(e => moment(e.startDt).hour())
-        ) - 1
-      : Math.min(...currentDateEntries.map(e => moment(e.startDt).hour())) - 1
-  );
-  const maxHour = Math.min(
-    24,
-    useWeekView
-      ? Math.max(
-          ...Object.values(entries)
-            .flat()
-            .map(e =>
+  const minHour = showAllTimeslots
+    ? 0
+    : Math.min(
+        eventStartDt.hour(),
+        ...(useWeekView
+          ? Object.values(entries)
+              .flat()
+              .map(e => e.startDt.hour())
+          : currentDateEntries.map(e => e.startDt.hour()))
+      );
```

Comment:
I'd go for one hour less, otherwise I don't see a way to move something one hour earlier.
```suggestion
  const minHour = showAllTimeslots
    ? 0
    : Math.max(
        Math.min(
          eventStartDt.hour(),
          ...(useWeekView
            ? Object.values(entries)
                .flat()
                .map(e => e.startDt.hour())
            : currentDateEntries.map(e => e.startDt.hour()))
        ) - 1,
        0
      );
```